circle.yml

```diff
@@ -11,6 +11,10 @@ general:
     - http-servers/build/libs
     - ssl-config/build/libs
 
+dependencies:
+  override:
+    - ./gradlew resolveDependencies
```

Comment:
Is there any code that exists for that, or is it something that's been planned/thought of but not yet written?